### PR TITLE
feat(sig): Submit proposal for a CI SIG

### PR DESCRIPTION
**Pull request by gal-yardeni**
Proposing a CI (continues integration) sig to align on current and future related CI work within Spinnaker.

### CODE REVIEW

**Comment by gal-yardeni** [comment]
Great idea @ezimanyi! It probably makes sense to add `igor` to this SIG, will talk to some colleagues and edit the proposal.

**Comment by gal-yardeni** [comment]
@emjburns , @ajordens Will love to hear your thoughts about the proposal - anything else missing? Thanks!

File: sig-ci/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* Gal Yardeni (https://github.com/gal-yardeni/) 
+
+## Contact
+
+* [Slack](https://spinnakerteam.slack.com/archives/CQJP8H05P)
+
+## Scope
+
+*Status and Alignment*
+  * `Igor` (https://github.com/spinnaker/igor)
```
Comment (robzienert) [inline sig-ci/README.md]:
```suggestion
  * [Igor](https://github.com/spinnaker/igor)
```

**Comment by gal-yardeni** [comment]
@robzienert  not yet - will do!

File: sig-ci/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# SIG: CI
+
+This SIG is focused on CI (continuous integration) experience within Spinnaker.
+
+## Meetings
+
+* Regular SIG Meeting
+  * Every other Tuesday, 1 PM EST / 10AM PST
```
Comment (dbyron0) [inline sig-ci/README.md]: Doesn't this conflict with either the kubernetes sig or the spinnaker as code sig?

Comment (ezimanyi) [reply sig-ci/README.md]: Yes this is the same time as the biweekly Kubernetes SIG.

Comment (plumpy) [reply sig-ci/README.md]: As long as it was on the opposite week (i.e. starting 1/7), it should be fine, though?

Comment (dbyron0) [reply sig-ci/README.md]: Aaah, right you are @plumpy .  Here's the calendar I've been looking at: https://calendar.google.com/calendar/embed?src=spinnaker.io_tr65rjf5mfij7p6vucprkhulcc%40group.calendar.google.com&ctz=America%2FLos_Angeles

Comment (gal-yardeni) [reply sig-ci/README.md]: Yep, we will start it in the opposite week from the Kubernetes sig, and we might change the date/time according to the meeting's participates as we go forward.

**Comment by gal-yardeni** [comment]
Tagging @jensvdh to review as well.

**Comment by ajordens** [comment]
At this point, I believe we're ready for @spinnaker/sc to ratify this new SIG.

File: sig-ci/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# SIG: CI
+
+This SIG is focused on CI (continuous integration) experience within Spinnaker.
```
Comment (demobox) [inline sig-ci/README.md]: Is this intended to be a long-lived SIG, or more of a project-focused group with the goal of delivering the CI view, defining necessary contracts for providers, etc.?

Comment (pstout) [reply sig-ci/README.md]: +1 to Andrew's question.

Comment (gal-yardeni) [reply sig-ci/README.md]: @demobox To your point, this sig is intended to be a long-lived sig and not just project-focused. We plan to tackle all CI elements within Spinnaker - Igor, CI view/config, potentially adding a CI stage, and so on. 
We will definitely collaborate with the UI/UX sig when we'll be considering adding elements to the UI such as the CI view.

Comment (aglover) [reply sig-ci/README.md]: Sounds exciting and thanks for the clarifications @gal-yardeni! Perhaps add that language in some way to the SIG description? 

+1 from me.

Comment (gal-yardeni) [reply sig-ci/README.md]: Done. Thanks @aglover!

Comment (demobox) [reply sig-ci/README.md]: Thanks for the additional details!

**Pull request merged**